src/main/menus/app.ts

```diff
@@ -149,14 +149,16 @@ export function createTemplate(config: Config, updateManager: UpdateManager) {
                 }
                 return 'Ctrl+Shift+I';
             })(),
-            click(item: Electron.MenuItem, focusedWindow?: BrowserWindow) {
-                if (focusedWindow) {
-                    // toggledevtools opens it in the last known position, so sometimes it goes below the browserview
-                    if (focusedWindow.webContents.isDevToolsOpened()) {
-                        focusedWindow.webContents.closeDevTools();
-                    } else {
-                        focusedWindow.webContents.openDevTools({mode: 'detach'});
-                    }
+            click() {
```

Comment:
This used to matter when we had the Settings and Main windows, but now we just have the main one so we can assume we're using that one.